tests/results/result_actions/ApplyPatchActionTest.py

```diff
@@ -156,14 +156,24 @@ def test_is_applicable_conflict(self):
 
         conflict_result = Result('', '', diffs={'f': diff})
         # Applying the same diff twice will result in a conflict
-        self.assertFalse(
-            ApplyPatchAction.is_applicable(conflict_result, {}, {'f': diff}))
+        self.assertIn(
+            'Two or more patches conflict with each other: ',
+            ApplyPatchAction.is_applicable(conflict_result, {}, {'f': diff})
+        )
 
     def test_is_applicable_empty_patch(self):
-        empty_patch_result = Result('', '', diffs={})
-        self.assertFalse(
-            ApplyPatchAction.is_applicable(empty_patch_result, {}, {}))
+        diff = Diff([], rename='new_name')
+        result = Result('', '', diffs={'f': diff})
+
+        # Two renames donot result in any change
+        self.assertEqual(
+            ApplyPatchAction.is_applicable(result, {'f': diff}, {'f': diff}),
```

Comment:
@sils `original_file_dict` is a dict containing filename as key and a diff as a value right?
it sure seems like it from is_applicable method of ShowPatchAction.